src/test/java/hw1/data/MultData_long.java

```diff
@@ -0,0 +1,16 @@
+package hw1.data;
+
+import org.testng.annotations.DataProvider;
+
+public class MultData_long {
+
+    @DataProvider(name = "MultDataProvider_long")
```

Comment:
According to Java convention, we should use camelcase in Java class names. Underscore symbol is suitable for private constants such as private static String MY_PRIVATE_VARIABLE